Fix global layout style & Update login layout

attitude opened the pull request:
Fixed layout:

<img width="732" alt="Screenshot 2022-01-24 at 16 37 29" src="https://user-images.githubusercontent.com/387611/150814516-0ac9344a-0585-4f51-9085-df44ecb8a3fb.png">

<img width="732" alt="Screenshot 2022-01-24 at 16 37 01" src="https://user-images.githubusercontent.com/387611/150814519-dc2f9895-6718-4801-be5b-c033933a63dc.png">


Closes https://github.com/contember/private-issues/issues/52

### Checks
- [x] The changes and implementation strategy have been consulted with the maintainers.
- [x] The code passes all the checks (Run `pnpm run check`).

JanTvrdik commented:
1) The error messages are too small and strangly styled.

![image](https://user-images.githubusercontent.com/175109/150817733-d8bbeb24-fdfa-439a-a5b1-45ff2f905aa2.png)


2)  Why is heading color blue when inputs are focused?

attitude commented:
I have fixed the password reset page layout (also the fill reset token).

1. I agree this the style of errors are not ideal. But lets fix them in a different PR because this is outside of the scope of this PR. [1]
2. Changing the box label to primary colour when focused is an accessibility hint and should remain. However I've changed how `MiscPageLayout` uses the box hence will not cause undesirable colour change.

I would suggest to fix what needs to be fixed right away (quick wins) and schedule next improvements without blocking the fixes. Maybe we should draw some design and discuss it first.

Is that acceptable way how to approach close this fix PR?

---
[1]: `ErrorList` historically uses `<Message />` component that is also being used on it's own in different context and I would need to make sure all other use cases are correct before proposing proper styling and I just haven't had a change to addressed all of it yet.

attitude commented:
@JanTvrdik please review the changes. I think I've fixed the outstanding issues. (Also please note my comment above.| Thanks!
